core/src/cubos/core/ui/serialization.cpp

```diff
@@ -269,12 +273,11 @@ static bool pickScalar(data::Package& pkg, const std::string& name)
         break;
     }
     case Type::String: {
-        char buf[1024];
-        memset(buf, 0, sizeof(buf));
-        strncpy(buf, pkg.get<std::string>().c_str(), sizeof(buf) - 1);
-        if (ImGui::InputText(name.c_str(), buf, sizeof(buf), ImGuiInputTextFlags_EnterReturnsTrue))
+        std::array<char, 1024> buf = {0};
```

Comment:
This doesn't initialize all elements to 0: it only sets the first element to the specified value, then the remaining elements get initialized to 0. So, it's "implicitely" working, but to avoid confusion I'd remove the `0`
```suggestion
        std::array<char, 1024> buf = {};
```